Project-II-Calculator/calculator/src/components/DisplayComponents/CalculatorDisplay.js

```diff
@@ -1,2 +1,146 @@
-import React from 'react';
-import './Display.css';
+import React from "react";
+import "./Display.css";
+
+import NumberButton from "../ButtonComponents/NumberButton";
+import ActionButton from "../ButtonComponents/ActionButton";
+export default function CalculatorDisplay() {
+  return (
+    <div className="calculatorDisplay">
+      <div className="labelDiv">
+        <label style={{ padding: "0 1rem" }}>0</label>
+      </div>
+
+      <div className="buttons">
+        <ActionButton
+          text="clear"
+          buttonStyle={{
+            backgroundColor: "transparent",
+            color: "black",
+            fontWeight: "normal"
+          }}
+        />
+
+        <NumberButton
+          text="÷"
+          buttonStyle={{
+            backgroundColor: "#a0001e",
+            color: "white"
+          }}
+        />
+
+        <NumberButton
+          text="7"
+          buttonStyle={{
+            backgroundColor: "transparent",
+            color: "black"
```

Comment:
This is working fine and you are passing down props correctly, however to inline the same style over and over is not really DRY. Why just don't pass a string as the prop and assign it to a className in your child component?